packages/subgraph/schema.graphql

```diff
@@ -1260,6 +1266,86 @@ type AccountTokenSnapshot @entity {
     account: Account!
     token: Token!
     flowOperators: [FlowOperator!]! @derivedFrom(field: "accountTokenSnapshot")
+    accountTokenSnapshotLogs: [AccountTokenSnapshotLog!]! @derivedFrom(field: "accountTokenSnapshot")
+}
+
+"""
+AccountTokenSnapshotLog: Historical entries of AccountTokenSnapshot updates.
+"""
+type AccountTokenSnapshotLog @entity {
```

Comment:
Shouldn't this be marked immutable somewhere? @0xdavinchee

Reply:
this is not super necessary tbh

Reply:
But it's supposed to be an immutable entity ;)

Reply:
let's add this to an issue and give the HOL entities an interface to implement and do this once the graph fixes the issue w/ mixing immutable/non-immutable

Reply:
Apparently it got fixed: https://github.com/graphprotocol/graph-node/issues/3553 ;) Maybe hasn't shown up in hosted service yet.